client/src/common/selectors.ts

```diff
@@ -7,9 +7,19 @@ import { FEATURES } from "../util/featureFlags/features";
 export function isSpatialMode(props: ShouldShowOpenseadragonProps): boolean {
   const { layoutChoice, panelEmbedding } = props;
 
-  return !!(
+  const { open, layoutChoice: panelEmbeddingLayoutChoice } =
+    panelEmbedding || {};
+
+  const isPanelEmbeddingInSpatialMode =
+    /**
+     * (thuang): If the side panel is not open, don't take its layout choice into account
+     */
+    open &&
```

Comment:
this is to resolve the bug when side panel is off, the image layer toggle button would still show up when the main panel is switched to a non-spatial embedding